### PR TITLE
Toolkit w/ templates for open source orgs writing sponsorship docs

### PR DESCRIPTION
**Pull request by amcasari**
New toolkit - short explanation + templates for open source organizations creating docs to communicate with potential + active sponsors.

Included are suggestions for:
- sponsorship prospectus
- event report (one-time sponsorship)
- annual report (active sponsor)

Open to all suggestions + feedback to make this most useful!

### CODE REVIEW

File: sponsorship-docs/sponsor-event-report.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  * Notable metrics (# of speakers, workshops, etc)
+* Sponsorship Impact
+  * List of benefits activated (from sponsorship contract/prospectus)
+  * What did this sponsorship enable for your community?
+
+
+## Example _open source community event report_
+
+**Organization:** Friends Rehabilitating Wild Bears in North America
+**Tax-status:** 5013c, registered in the United States
```
Comment (emckean) [inline sponsorship-docs/sponsor-event-report.md]: do you mean 501(c)3?

File: sponsorship-docs/sponsor-prospectus.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* What does each option/level enable for the organization?
+* What does each option/level provide as sponsorship benefits?
+* Who does this serve (open source communities, projects, etc)?
+* What open source software (project, portfolio, foundation) benefits from this sponsorship?
+* Is there a Code of Conduct in place for this (organization, community, event)?
+
+## Example sponsorship prospectus
+
+**Organization:** Friends Rehabilitating Wild Bears in North America
+**Tax-status:** 5013c, registered in the United States
```
Comment (emckean) [inline sponsorship-docs/sponsor-prospectus.md]: 501(c)3 :)

File: sponsorship-docs/sponsor-event-report.md (hunk 1 above)
Comment (amcasari) [reply sponsorship-docs/sponsor-event-report.md]: Should be fixed now!

File: sponsorship-docs/sponsor-prospectus.md (hunk 2 above)
Comment (amcasari) [reply sponsorship-docs/sponsor-prospectus.md]: Should be fixed now!

**Pull request merged**